docs/OTAnalytics/usage-cli.md

```diff
@@ -0,0 +1,234 @@
+# Usage CLI
+
+## Synopsis
+
+```text
+python -m OTAnalytics [-h] [--cli] [--cli-mode {bulk,stream}]
+                [--cli-chunk-size CLI_CHUNK_SIZE] [--show-svz]
+                [--config CONFIG] [--ottrks OTTRKS [OTTRKS ...]]
+                [--otflow OTFLOW] [--save-dir SAVE_DIR]
+                [--save-name SAVE_NAME] [--save-suffix SAVE_SUFFIX]
+                [--debug]
+                [--event-formats EVENT_FORMATS [EVENT_FORMATS ...]]
+                [--count-intervals COUNT_INTERVALS [COUNT_INTERVALS ...]]
+                [--no-track-export] [--no-track-statistics-export]
+                [--num-processes NUM_PROCESSES] [--logfile LOGFILE]
+                [--logfile_overwrite]
+                [--include-classes INCLUDE_CLASSES [INCLUDE_CLASSES ...]]
+                [--exclude-classes EXCLUDE_CLASSES [EXCLUDE_CLASSES ...]]
+```
+
+## Description
+
+If you have run [OTVision track](../../OTVision/usage/track)
+to extract movements of road users,
+you are ready to analyze the movements in each file.
+
```

Comment:
Are there any differences between processing with cli or gui? cli does not respect vehicles occurring in two ottrk files, does it?

Reply:
The cli does the same analysis as the gui. Thus, it also reads multiple ottrk files if configured.